app/views/app/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>New Post</h1>
+
+<%= form_with model: @post do |form| %>
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
```

Comment:
* As requested by the previous reviewer The user views related to the users' controller are missing and
 The post views related to post controllers are missing, Kindly add view files for each of the controller(posts, users) actions(index, show) [For refresher check project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=For%20each%20URL,the%20following%20output%3A)